NYPLAudiobookToolkit/Bookmarks/NYPLAudiobookBookmarks.swift

```diff
@@ -0,0 +1,91 @@
+import Foundation
+import NYPLUtilities
+
+public protocol NYPLAudiobookBookmarksBusinessLogicDelegate {
```

Comment:
maybe this should be moved to the file that contains the business logic class? since it's not used here...

Reply:
This will be used when UI being implement, and the business logic needs to stay in Simplified-iOS because it needs access to NYPLAnnotations.swift

Reply:
oh sorry, I didn't mean to move it out of the repo. For some reason I thought the business logic was in the same repo. Maybe it can be its own file?

Reply:
Sure